src/containers/Cinema/Cinema.tsx

```diff
@@ -1,192 +1,147 @@
-
-      // we already loaded this item
-      if (currentItem && currentItem.mediaid === item.mediaid) {
-        return;
-      }
-      // load new item
-      playerRef.current.load([deepCopy(item)]);
-      calculateWatchHistoryProgress();
-    };
-
-    const initializePlayer = () => {
-      if (!window.jwplayer || !playerElementRef.current) return;
-
-      playerRef.current = window.jwplayer(playerElementRef.current) as JWPlayer;
-
-      playerRef.current.setup({
-        playlist: [deepCopy(item)],
-        aspect: false,
-        width: '100%',
-        height: '100%',
-        mute: false,
-        autostart: true,
-        repeat: false,
-      });
-
-      attachEvents();
-      calculateWatchHistoryProgress();
-      setPlayer(playerRef.current);
-
-      playerRef.current.setPlaylistItemCallback(handlePlaylistItemCallback);
-    };
-
-    if (playerRef.current) {
-      return loadPlaylist();
+    if (open) {
+      setUserActive(true);
+      document.body.style.overflowY = 'hidden';
+    } else {
+      saveItem(item, getProgress());
     }
 
-    if (libLoaded) {
-      initializePlayer();
-    }
-  }, [libLoaded, item, enableWatchHistory, setPlayer, handlePlaylistItemCallback, detachEvents, attachEvents, player]);
-
-  useEffect(() => {
     return () => {
-      if (playerRef.current) {
-        // Detaching events before component unmount
-        detachEvents();
-        playerRef.current.remove();
-      }
+      document.body.style.overflowY = '';
     };
-  }, [detachEvents]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [open]);
 
   return (
-    <div className={classNames(styles.cinema, { [styles.fill]: !isTrailer })}>
-      <div ref={playerElementRef} />
-    </div>
+    <Fade open={open}>
+      <div className={styles.playerContainer}>
+        <div className={styles.player}>
+          <Player
+            playerId={player}
+            feedId={feedId}
+            item={item}
+            onReady={handleReady}
+            onPlay={handlePlay}
+            onPause={handlePause}
+            onComplete={handleComplete}
+            onUserActive={handleUserActive}
+            onUserInActive={handleUserInactive}
+            onPlaylistItemCallback={handlePlaylistItemCallback}
+            startTime={startTime}
+          />
+        </div>
+        <Fade open={!isPlaying || userActive}>
+          <div className={styles.playerOverlay}>
+            <div className={styles.playerContent}>
+              <IconButton aria-label={t('common:back')} onClick={onClose} className={styles.backButton}>
+                <ArrowLeft />
+              </IconButton>
+              <div>
+                <h2 className={styles.title}>{title}</h2>
+                <div className={styles.metaContainer}>
+                  {isSeries && (
+                    <div className={styles.seriesMeta}>
+                      <strong>{seriesMeta}</strong> - {episodeTitle}
+                    </div>
+                  )}
```

Comment:
Do you need the isSeries property or can you just use the truthiness of seriesMeta to decide to show it?

```suggestion
                  {seriesMeta && (
                    <div className={styles.seriesMeta}>
                      <strong>{seriesMeta}</strong> - {episodeTitle}
                    </div>
                  )}
```

Reply:
Done (using the `secondaryMetadata` prop)